index.bs

```diff
@@ -11147,7 +11157,15 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 
 			2. Remove the [=oldest message=] of <var>Q<sub>rendering</sub></var>.
 
-	3. Process a render quantum.
+
+	3. Process the regular event queue.
+
+		1. Let <var>task count</var> be the number of tasks in the task queue of
+			the task queue of the {{AudioContext}} being rendered.
+
+		2. Spin the event loop until <var>task count</var> tasks have been executed.
```

Comment:
Do we really need `task count`? Can we just say:

"Run the event loop until all queued tasks in _Q_ are executed."

(_Q_ can be defined somewhere)

Reply:
I haven't found a way to define this more cleanly, this is unambiguous. In particular, I haven't found a way to define `Q`. I'm happy to hear any idea though.